src/UserInterfaceLayer.cpp

```diff
@@ -21,28 +26,32 @@ void UserInterfaceLayer::Start() {
     init_pair(4, COLOR_MAGENTA, COLOR_BLACK);
     init_pair(5, COLOR_WHITE, COLOR_BLACK);
 
-    int row, col;
-    getmaxyx(stdscr, row, col);
+    init_pair(6, COLOR_WHITE, COLOR_GREEN);
+    init_pair(7, COLOR_WHITE, COLOR_YELLOW);
+    init_pair(8, COLOR_WHITE, COLOR_CYAN);
+    init_pair(9, COLOR_WHITE, COLOR_MAGENTA);
```

Comment:
Not sure what's going on here...?

Reply:
Not used for now, we can define as we go i think.